rest-api/Dockerfile

```diff
@@ -40,27 +40,27 @@ ADD ./src/main/resources/docker/start_rest_api.sh start_rest_api.sh
 RUN chmod +x start_rest_api.sh && \
     rm -rf webapps/*
 
-ADD ./target/${WAR_FILE} webapps/ROOT.war
+ADD ./target/rest-api-3.0.0-SNAPSHOT.war webapps/ROOT.war
```

Comment:
why not to keep it as a parameter? Does this mean that we'll have to change this string on each release?

Reply:
You are right, my mistake.